nixos/modules/services/backup/burpClient.nix

```diff
@@ -0,0 +1,65 @@
+{ config, lib, networking, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.burpClient;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.burpClient = {
+
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable burp.";
+      };
+
+      configFile = mkOption {
+        default = "/etc/burp/burp.conf";
```

Comment:
We try to avoid files in `/etc` if possible. Please use technique like here: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/computing/slurm/slurm.nix#L9.
Write a config file and pass a path to `burp`.
You can create an option like `services.burpClient.config`. And then use
```

cfg = config.services.burpClient;
writeText "burp.conf" ''
  ${cfg.config}
''
```